test/Mvc6Framework45.FunctionalTests/project.json

```diff
@@ -1,50 +1,62 @@
 {
   "webroot": "wwwroot",
   "userSecretsId": "aspnet5-Mvc6Framework45.FunctionalTests-42f45116-363a-40d3-9c44-bc58885d6e96",
-  "version": "1.0.0-rc1-*",
+  "version": "1.0.0-rc2",
+  "compilationOptions": {
+    "preserveCompilationContext": true
+  },
   "dependencies": {
-    "EntityFramework.MicrosoftSqlServer": "7.0.0-rc1-final",
```

Comment:
@davidfowl I have removed netstandard library from the dependencies in AI.AspnetCore. We still see "TypeLoadExceptions"/"FileNotFoundExceptions" when we add ApplicationInsights Nuget (the one we generate from above) to a new MVC application (created using Visual studio CLI tools, with DnxCore50 framework). Please let me know if we are missing anything with respect to that setup. Thanks!